ports/rp2/boards/SPARKFUN_XRP_CONTROLLER_BETA/pins.csv

```diff
@@ -0,0 +1,93 @@
+# All GPIO pins available to user
+GP0,GPIO0
+GP1,GPIO1
+GP2,GPIO2
+GP3,GPIO3
+GP4,GPIO4
+GP5,GPIO5
+GP6,GPIO6
+GP7,GPIO7
+GP8,GPIO8
+GP9,GPIO9
+GP10,GPIO10
+GP11,GPIO11
+GP12,GPIO12
+GP13,GPIO13
+GP14,GPIO14
+GP15,GPIO15
+GP16,GPIO16
+GP17,GPIO17
+GP18,GPIO18
+GP19,GPIO19
+GP20,GPIO20
+GP21,GPIO21
+GP22,GPIO22
+GP26,GPIO26
+GP27,GPIO27
+GP28,GPIO28
+
+# Radio GPIO pins
+WL_GPIO0,EXT_GPIO0
+WL_GPIO1,EXT_GPIO1
+WL_GPIO2,EXT_GPIO2
+
+# LED default names
+LED,EXT_GPIO0
+
+# XRP default pin names
+MOTOR_L_IN_1,GPIO6
+MOTOR_L_IN_2,GPIO7
+MOTOR_R_IN_1,GPIO14
+MOTOR_R_IN_2,GPIO15
+MOTOR_3_IN_1,GPIO2
+MOTOR_3_IN_2,GPIO3
+MOTOR_4_IN_1,GPIO10
+MOTOR_4_IN_2,GPIO11
+MOTOR_L_ENCODER_A,GPIO4
+MOTOR_L_ENCODER_B,GPIO5
+MOTOR_R_ENCODER_A,GPIO12
+MOTOR_R_ENCODER_B,GPIO13
+MOTOR_3_ENCODER_A,GPIO0
+MOTOR_3_ENCODER_B,GPIO1
+MOTOR_4_ENCODER_A,GPIO8
+MOTOR_4_ENCODER_B,GPIO9
+SERVO_1,GPIO16
+SERVO_2,GPIO17
+I2C_SDA_1,GPIO18
+I2C_SCL_1,GPIO19
```

Comment:
If `I2C(1)` is available on these pins, you probably want to add the following to `mpconfigboard.h`:
```c
#define MICROPY_HW_I2C1_SCL (19)
#define MICROPY_HW_I2C1_SDA (18)
```
Then users can just do `machine.I2C(1)`, without specifying the pins.

Reply:
Good catch, fixed!